pages/linux/genid.md

```diff
@@ -0,0 +1,24 @@
+# genid
+
+> Generate IDs, such as snowflakes, UUIDs, and a new GAID.
+> More information: <https://github.com/bleonard252/genid>.
+
+- Generate a UUIDv4:
```

Comment:
In #5892, we use `v4 UUID` instead of `UUIDv4`. I don't know which is better, but we should unify it. (cc @peterbabic)

Reply:
@marchersimon agreed, but [the official standard](https://tools.ietf.org/html/rfc4122) mentions it in neither short form but rather in the verbose form of `version 1 UUID` or in some cases `UUID version 1`. Maybe we should follow that. What do you think?

Reply:
I'm not sure if using the long form is necessary, since it's pretty long and everybody knows what `v1` means. Personally I like `UUIDv1` (probably because of `IPv4` and `IPv6`).